target/linux/ath79/base-files/etc/board.d/02_network

```diff
@@ -65,6 +65,12 @@ ath79_setup_interfaces()
 		ucidef_add_switch "switch0" \
 			"0@eth0" "1:lan:1" "3:lan:4" "4:lan:3" "5:lan:2" "2:wan"
 		;;
+	comfast,cf-wr650ac-v1|\
+        comfast,cf-wr650ac-v2)
```

Comment:
Wrong whitespace characters here.
Despite, please move this block down to account for alphabetical sorting.

Reply:
fixed

Reply:
fixed